### PR TITLE
Add TerminalStress, Mike Treit's application for breaking WT

### PR DESCRIPTION
**Pull request by DHowett**
From Treit/TerminalStress@39c03e2d008472cea039751bc54e12eff2ac6fc7

Co-authored-by: Dustin L. Howett <duhowett@microsoft.com>

### CODE REVIEW

**Comment by DHowett** [comment]
@Treit, thanks for permission to do this :smile:

File: src/tools/TerminalStress/TerminalStress.csproj [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net6.0</TargetFramework>
```
Comment (DHowett) [inline src/tools/TerminalStress/TerminalStress.csproj:5]: This is different from upstream, but I'm doing it to keep our package versions across the repo in sync

**Review by lhecker — APPROVED** [review]

**Comment by lhecker** [comment]
I think the Co-authored-by should mention Mike and not you, right? Because you're already the commit author anyways?

**Comment by DHowett** [comment]
> I think the Co-authored-by should mention Mike and not you, right? Because you're already the commit author anyways?

The other way around - Mike is the Author, I am the Committer, and I contributed the solution etc

**Comment by DHowett** [comment]
image

**Review by zadjii-msft — APPROVED** [review]
It's actually less aggressive than I expected, tbh.

**Review by carlos-zamora — APPROVED** [review]

**Pull request merged**